frontend/src/languages/french.js

```diff
@@ -5,7 +5,8 @@ const fr = {
       decimal: ",",
       thousand: ".",
       precision: 2
-    }
+    },
+    numberRegex: /^([0-9]{1,3}.([0-9]{3}.)*[0-9]{3}|[0-9]+)(,[0-9]+)?$/
```

Comment:
describe regex in a comment

Reply:
comments:
    // numberRegex describes values with "." as decimal seperator (matches e.g. 1000; 1,000; 1000.00; 1,000.00)
or
    // numberRegex describes values with "," as decimal seperator (matches e.g. 1000; 1.000; 1000,00; 1.000,00)